Lib/http/cookiejar.py

```diff
@@ -533,14 +533,20 @@ def parse_ns_headers(ns_headers):
 
 
 IPV4_RE = re.compile(r"\.\d+$", re.ASCII)
```

Comment:
I'm tempted to remove this attribute but it could be used in legacy code so add a comment "kept for backwards compatibility".